src/components/CardComponent.js

```diff
@@ -1,16 +1,25 @@
-/** display user information in a card */
-const CardComponent = ({members:{img,name,city,company,designation}}) =>{
+import {Link} from "react-router-dom"
```

Comment:
This is a generic comment. I generally follow a principle of keeping related files in a folder. So you can club all the similar files (eg: component.js and component.css files in a single folder) i.e. A folder for each component that you make. So later if the component has a lot of styles, or if you add a lot of new components, especially if you have to write test case for each component, then it'll be much easier to segregate .